web/src/main/java/se/inera/intyg/webcert/web/web/controller/integration/IntygIntegrationController.java

```diff
@@ -64,7 +64,7 @@
  * @author bensam
  */
 @Path("/intyg")
-@Api(value = "intyg (Djupintegration)", description = "REST API för Djupintegration", produces = MediaType.APPLICATION_JSON)
+@Api(value = "intyg (Dj upintegration)", description = "REST API för Djupintegration", produces = MediaType.APPLICATION_JSON)
```

Comment:
Ser ut som det kommit in ett extra space här.

Reply:
Fixat